web/crux-ui/src/elements/dyo-single-file-upload.tsx

```diff
@@ -0,0 +1,47 @@
+import useTranslation from 'next-translate/useTranslation'
+import React from 'react'
+import { DyoButton } from './dyo-button'
+import { DyoLabel } from './dyo-label'
+
+export interface DyoSingleFileUploadProps {
+  name: string
+  label?: string
+  handleFile: Function
+  accept: string
+}
+
+export const DyoSingleFileUploadInput = (props: DyoSingleFileUploadProps) => {
+  const { t } = useTranslation('common')
+  const { label, accept, handleFile } = props
+
+  const hiddenFileInput = React.useRef(null)
+
+  const handleClick = () => {
+    hiddenFileInput.current.click()
+  }
+
+  const handleChange = event => {
+    const fileUploaded = event.target.files[0]
```

Comment:
Why have to pick the first element in the files array? Can I upload multiple files?

Reply:
At the moment you cant upload multiple files, but as I mentioned in my previous comment I will refactor the component

Reply:
Okey, thanks.